main.cpp

```diff
@@ -102,6 +102,11 @@ int main(int, char**)
 
     Lamp::Games::getInstance().currentProfile = Lamp::Games::getInstance().currentGame->KeyInfo()["CurrentProfile"];
 
+    // Try to load and set the global font scale. I am pretty sure this is not a good way of doing this...
+    std::string loadedFontScale = Lamp::Core::FS::lampIO::loadKeyData("Font_Scale", "LAMP CONFIG");
```

Comment:
I agree its not the most efficient, but it works well enough, although id love to discuss ways it could be improved. 

On the matter of fontscale it is currently the only method to set text size unless you implement the Markdown Renderer or add a custom font and set its size during creation. While lampray does *technically* support custom fonts its set to the standard size.